internal/winapi/filesystem.go

```diff
@@ -59,3 +96,15 @@ type FileLinkInformation struct {
 	FileNameLength  uint32
 	FileName        [1]uint16
 }
+
+// C declaration:
+//   typedef struct _FILE_ID_INFO {
+//       ULONGLONG   VolumeSerialNumber;
+//       FILE_ID_128 FileId;
```

Comment:
nit: Could include a link to this definition also but just saving one extra click really. https://docs.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-file_id_info

Reply:
This is the same link I have 3 lines down?

Reply:
If you're talking about a link for `FILE_ID_128`, I don't think it's worthwhile to link to every nested struct definition.

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-file_id_128 Yep, pasted the wrong link.

Reply:
And that's fair